packages/devtools_app/lib/src/theme.dart

```diff
@@ -201,6 +201,45 @@ extension DevToolsColorScheme on ColorScheme {
       isLight ? const Color(0xFFB20001) : const Color(0xFFD88E73);
   Color get numericConstantSyntaxColor =>
       isLight ? const Color(0xFF098658) : const Color(0xFFB5CEA8);
+
+  // Light theme hover background is semi-transparent YellowAccent[100].
+  Color get hoverBackgroundColor => const Color.fromARGB(150, 255, 255, 141);
+
+  // Bar color for current selection (hover).
+  Color get hoverSelectionBarColor =>
+      isLight ? Colors.lime[600] : Colors.yellowAccent;
+
+  // Title of the hover card.
+  TextStyle get hoverTitleTextStyle => const TextStyle(
+        color: Colors.black,
+        fontWeight: FontWeight.w600,
```

Comment:
I would use bold not semibold to reduce the # of different colors we use.

Reply:
Done.